plugin/integration_test.go

```diff
@@ -87,11 +87,11 @@ func TestIntegration(t *testing.T) {
 	}
 
 	// validate and create
-	plugin, err := mgr.Get(context.Background(), "foo")
+	pl, err := mgr.Get(context.Background(), "foo")
 	if err != nil {
 		t.Fatal(err)
 	}
-	metadata, err := plugin.GetMetadata(context.Background(), &proto.GetMetadataRequest{})
+	metadata, err := pl.GetMetadata(context.Background(), &plugin.GetMetadataRequest{})
```

Comment:
Should we keep `proto.GetMetadataRequest{}` for unit test

Reply:
this is integration test. We have kept proto for unit tests.